tools/build_pytorch_libs.py

```diff
@@ -71,7 +71,7 @@ def cmake_defines(lst, **kwargs):
 # Ninja
 # Use ninja if it is on the PATH. Previous version of PyTorch required the
 # ninja python package, but we no longer use it, so we do not have to import it
-USE_NINJA = which('ninja') is not None
+USE_NINJA = USE_NINJA and (which('ninja') is not None)
```

Comment:
nit: small preference to put it all on one line:

```
USE_NINJA = not check_negative_env_flag('USE_NINJA') and which('ninja') is not None
```

As it is, the setting of USE_NINJA is far away from its use. And `USE_NINJA` use anywhere outside of build_pytorch_libs won't be in alignment with USE_NINJA inside of it.

Reply:
Sure